product details Show on Users clicking time of product #30

Sureshwebdeveloper opened the pull request:
product details on clicking time of product #30

## Description
This pull request fixes the User Click Time Show Product Details 

## Related Issue
Closes #30 

https://github.com/user-attachments/assets/c407b3cb-c580-44ae-8556-e131aed1d781

Sureshwebdeveloper committed:
product deatiles on clicking time of product #30

Sureshwebdeveloper commented:
@ChandelAnish please check above product click time show deatiles from back end to front end if any changes need to change please tell what I will do ?

ChandelAnish reviewed (commented):
-->onclicking on the bag icon in the product card product details page shows up that should not happen onclicking on the bag icon item should be added to the bag only.

-->add to cart button should work
-->PR on the dev branch


https://github.com/user-attachments/assets/aef7418c-9619-4c6a-bc4a-ee20275d67b4

Sureshwebdeveloper commented:
Sure Bro Are You Saying Need To Add A Cart Functionality When Ever User Clicks Cart Button The Item Should Add To Cart Page Right ? If dont have item the bag shouldn't open ? Please Dont Decrease a Level  I have Worked a lot I have Complete And make Pull Request On As Soon as possible  🙂
@ChandelAnish

Sureshwebdeveloper committed:
add to bag feature added

Sureshwebdeveloper commented:
> -->onclicking on the bag icon in the product card product details page shows up that should not happen onclicking on the bag icon item should be added to the bag only.
> 
> -->add to cart button should work -->PR on the dev branch
> 
>  Screen.Recording.2024-10-12.164300.mp4

https://github.com/user-attachments/assets/ed7c0e1f-949b-4d82-a995-9e0bb2132131

I have added to cart feature on my latest commit please check Above video...
please dont decrease level first you have assigned me and added level 3 now the level is true please add  a level 3
closes #30  @ChandelAnish

Sureshwebdeveloper committed:
Merge branch 'ChandelAnish:master' into master

Sureshwebdeveloper commented:
@ChandelAnish  please check and merge
